src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir, sans-serif;
```

Comment:
`sans-serif` is odd here

Reply:
[CHECKLIST](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) point 7